test/test_pointnav_dataset.py

```diff
@@ -109,6 +109,54 @@ def test_multiple_files_pointnav_dataset():
     check_json_serializaiton(partial_dataset)
 
 
+@pytest.mark.parametrize("split", ["train", "val"])
+def test_dataset_splitting(split):
```

Comment:
If that makes sense to have one test for dataset configs as params. Then we can make sure new tasks will be just added.

Reply:
Different datasets will have different splits available and will be different sizes.  For PointNav and ObjectNav, it only makes sense to load a few scenes and split on those, while for EQA and VLN, it is feasible to load all of train and split that, so that singular test will likely end up having way too many flags and conditions and people will have a harder time adding to it.

Reply:
Got it, thank you for the clarification.